instrumentation/struts-2.3/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    `java-library`
+    id("io.opentelemetry.instrumentation.auto-instrumentation")
+}
+
+repositories {
+    mavenCentral()
+}
+val versions: Map<String, String> by extra
```

Comment:
please add an empty line in between different statements, it helps with readability.